src/tasks/keycloak/config.ts

```diff
@@ -241,6 +241,21 @@ export const otomiClientCfgTpl = (
   id: 'otomi',
   secret,
   defaultClientScopes: ['openid', 'email', 'profile'],
+  protocolMappers: [
```

Comment:
This is interesting.
Why new mapper needs to be added to keycloak?
Could you add some comments in code.